src/vm/mngstdinterfaces.cpp

```diff
@@ -990,9 +990,7 @@ FCIMPL1(Object*, StdMngIEnumerable::GetEnumerator, Object* refThisUNSAFE)
 
     if (retVal == NULL)
     {
-        // In desktop CLR we'll attempt to call through IDispatch(DISPID_NEWENUM)
-        // This is not supported in CoreCLR
-        COMPlusThrow(kPlatformNotSupportedException, IDS_EE_ERROR_IDISPATCH);
```

Comment:
Please make sure to remove `IDS_EE_ERROR_IDISPATCH` if it isn't used anywhere else.